src/scss/components/_buttons.scss

```diff
@@ -31,7 +30,8 @@
   &.usa-button--active,
   &:disabled,
   &.usa-button--disabled {
-    @include no-knockout-font-smoothing;
+    -moz-osx-font-smoothing: inherit;
+    -webkit-font-smoothing: inherit;
```

Comment:
These styles only apply for hover, active, and disabled 🤦  Should have also duplicated above, after the `@include button-unstyled`. Was misled by the fact that this is substituting the behavior of `no-knockout-font-smoothing`.